quick-start/QUICK_START.md

```
@@ -0,0 +1,123 @@
+./bootstrap.sh --skip-kind --skip-openchoreo
+```
+
+**Time:** ~5-8 minutes
+
+This installs the Agent Management Platform on your existing OpenChoreo cluster.
+
+## Access Your Platform
+
+After installation completes, your platform is automatically accessible at:
+
+- **Console**: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)